src/components/PageSwitcher/components/EEGEduEvoked/EEGEduEvoked.js

```diff
@@ -0,0 +1,300 @@
+    console.log("Subscribed to Evoked");
+  }
+}
+
+export function renderModule(channels) {
+  function renderCharts() { 
+      return null
+  }
+
+  return (
+    <Card title={specificTranslations.title}>
+      <Card.Section>
+        <Stack>
+          <TextContainer>
+            <p>{specificTranslations.description}</p>
+          </TextContainer>
+        </Stack>
+      </Card.Section>
+      <Card.Section>
+        <div style={chartStyles.wrapperStyle.style}>{renderCharts()}</div>
+      </Card.Section>
+    </Card>
+  );
+}
+  
+export function renderSliders(setData, setSettings, status, Settings) {
+
+  function resetPipeSetup(value) {
+    buildPipe(Settings);
+    setup(setData, Settings)
+  }
+  
+  function handleIntervalRangeSliderChange(value) {
+    setSettings(prevState => ({...prevState, interval: value}));
+    resetPipeSetup();
+  }
+
+  function handleCutoffLowRangeSliderChange(value) {
+    setSettings(prevState => ({...prevState, cutOffLow: value}));
+    resetPipeSetup();
+  }
+
+  function handleCutoffHighRangeSliderChange(value) {
+    setSettings(prevState => ({...prevState, cutOffHigh: value}));
+    resetPipeSetup();
+ }
+
+  function handleDurationRangeSliderChange(value) {
+    setSettings(prevState => ({...prevState, duration: value}));
+    resetPipeSetup();
+ }
+
+  return (
+    <Card title={Settings.name + ' Settings'} sectioned>
+      <RangeSlider 
+        disabled={status === generalTranslations.connect} 
+        min={128} step={128} max={4096}
+        label={'Epoch duration (Sampling Points): ' + Settings.duration} 
+        value={Settings.duration} 
+        onChange={handleDurationRangeSliderChange} 
+      />          
+      <RangeSlider 
+        disabled={status === generalTranslations.connect} 
+        min={10} step={1} max={Settings.duration}
+        label={'Sampling points between epochs onsets: ' + Settings.interval} 
+        value={Settings.interval} 
+        onChange={handleIntervalRangeSliderChange} 
+      />
+      <RangeSlider 
+        disabled={status === generalTranslations.connect} 
+        min={.01} step={.5} max={Settings.cutOffHigh - .5}
+        label={'Cutoff Frequency Low: ' + Settings.cutOffLow + ' Hz'} 
+        value={Settings.cutOffLow} 
+        onChange={handleCutoffLowRangeSliderChange} 
+      />
+      <RangeSlider 
+        disabled={status === generalTranslations.connect} 
+        min={Settings.cutOffLow + .5} step={.5} max={Settings.srate/2}
+        label={'Cutoff Frequency High: ' + Settings.cutOffHigh + ' Hz'} 
+        value={Settings.cutOffHigh} 
+        onChange={handleCutoffHighRangeSliderChange} 
+      />
+    </Card>
+  )
+}
+
+export function renderRecord(recordPopChange, recordPop, status, Settings) {
+  return (
+    <Card title={'Run ERP experiment'} sectioned>
+      <Card.Section>
+        <p>
+          {"Clicking this button will begin the experiment so check your data quality on the raw module first. "}
```

Comment:
lots of explaination here for what will happen, and an example of what data will look like, we should add this this semester for the other modules as well